src/EditorFeatures/Core.Wpf/Structure/AbstractStructureTaggerProvider.cs

```diff
@@ -126,51 +128,29 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView textVie
         {
             try
             {
+                var document = documentSnapshotSpan.Document;
```

Comment:
Can we get this null annotated to avoid this surprise in the future?

Reply:
Done.